test/cases/basic_test.rb

```diff
@@ -146,7 +147,7 @@ def test_spatial_factory_attrs_parsing
                                                 has_z: false, has_m: false })
 
       # wrong factory for default
-      old_default = spatial_factory_store.default
+      old_default = spatial_factory_store.instance_variable_get :@default
```

Comment:
I'm not sure this is an ok behaviour in the lib (I mean if it is not clear when testing it, it may not be clear when using it), you've just work on it @formigarafa WDYT?

Reply:
I did not like that either but there is no other way to infer if there is a default set in the factory store.

From `rgeo-ativerecord`: https://github.com/rgeo/rgeo-activerecord/blob/master/lib/rgeo/active_record/spatial_factory_store.rb#L20-L22
```ruby
      def default(attrs = {})
        @default || default_for_attrs(attrs)
      end
```
Maybe If this method used `||=` then it would work as the code seem to expect.

Reply:
I'm unfortunately not sure it is that simple, for instance with `||=` you'd get a weird behaviour doing:

```ruby
f1 = store.default(some_attr)
f2 = stor.default(other_attr)
```

both would be the same, ignoring `other_attr`.

I opened an issue to see if there is a good way to refactor this https://github.com/rgeo/rgeo-activerecord/issues/80